core/vm/errors.go

```diff
@@ -36,6 +36,7 @@ var (
 	ErrGasUintOverflow          = errors.New("gas uint64 overflow")
 	ErrInvalidCode              = errors.New("invalid code: must not begin with 0xef")
 	ErrNonceUintOverflow        = errors.New("nonce uint64 overflow")
+	ErrInvalidOpcode            = errors.New("aborted with INVALID (0xfe) opcode")
```

Comment:
We already have an `ErrInvalidOpCode`, as a `type`, defined further down in this file. Having a type with the same name as a global `var` is not a good idea. 

IMO let's not define a new specific error for this specific opcode -- the idea with `0xFE` is that it will always remain undefined. There are others _currently_ undefined, and IMO this opcode should be treated exactly like those, by the EVM. 

So I think we can skip this `var` here.   And the `opInvaid` could just do 
```
return nil, &ErrInvalidOpCode{opcode: OpCode(0xfe)}
```